routes/api.php

```diff
@@ -16,3 +16,5 @@
 
     Route::get('v1/search', [SearchController::class, 'search']);
 });
+
+Route::post('/v1/auth/login', [AuthController::class, 'login']);
```

Comment:
add this into the function for congruity remember to remove the v1 part as its prefixed already